contributing/README.md

```diff
@@ -34,6 +34,15 @@ Start the title with `[ComponentName]` to identify which component a change affe
 [FooBar] Removes the deprecated fooWithBar:(Bar*)bar method.
 ```
 
+### Pull request continuous integration for new contributors
+
+Continuous integration will only be initiated automatically for the core team and
```

Comment:
I assume this is [`core-ios-team`](https://github.com/orgs/material-components/teams/core-ios-team) and not [`mdc-ios`](https://github.com/orgs/material-components/teams/mdc-ios)? Could you please link to the explicit team so readers know we're using these terms as proper nouns and not general concepts?